plone/app/blocks/layoutbehavior.py

```diff
@@ -244,17 +256,13 @@ def sectionSiteLayout(self, value):
     def tile_layout(self):
         return self.content or u''
 
+    def content_layout_path(self):
+        path = self.contentLayout
+        return path or super(LayoutAwareBehavior, self).content_layout_path()
+
     def content_layout(self):
-        if self.contentLayout:
-            try:
-                path = self.contentLayout
-                resolved = resolveResource(path)
-                return applyTilePersistent(path, resolved)
-            except (NotFound, RuntimeError, IOError):
-                pass
-        elif self.customContentLayout:
+        if self.customContentLayout and not self.contentLayout:
```

Comment:
@datakurre - I recognized while refactoring - shouldn't customContentLayout be preferred over self.contentLayout? Otherwise customizations of the content layout are likely not applied.